http-db-service/handler/utils/utils.go

```diff
@@ -0,0 +1,21 @@
+package utils
```

Comment:
In general, packages such as utils, common, helpers, base, etc... are considered bad practice in Go.

My suggestion would be having a more specific package such as `response`; then the exported function would be `response.WithCodeAndMessage()`.

Furthermore, I would unexport the `ErrorResponse` struct and provide a function to get the status and message (e.g. `response.GetCodeAndMessage()`).

Reply:
Applied the review comments after discussion with @clebs. Please have a look.